modules/Sources/Features/SecurityWarning/SecurityWarningStore.swift

```diff
@@ -18,43 +18,37 @@ import Generated
 import Utils
 import SwiftUI
 
-public typealias SecurityWarningStore = Store<SecurityWarningReducer.State, SecurityWarningReducer.Action>
-public typealias SecurityWarningViewStore = ViewStore<SecurityWarningReducer.State, SecurityWarningReducer.Action>
-
-public struct SecurityWarningReducer: Reducer {
+@Reducer
+public struct SecurityWarning {
+    @ObservableState
     public struct State: Equatable {
-        public enum Destination: Equatable, CaseIterable {
-            case createNewWallet
-        }
-
-        @PresentationState public var alert: AlertState<Action>?
+        @Presents public var alert: AlertState<Action>?
         public var appBuild = ""
         public var appVersion = ""
-        public var destination: Destination?
-        @BindingState public var isAcknowledged: Bool = false
+        public var isAcknowledged: Bool = false
+        public var recoveryPhraseDisplayBinding: Bool = false
```

Comment:
I don't understand what this new variable is for; it is not documented anywhere, and the UI screenshots do not include anything that looks like it might display a recovery phrase.

Reply:
This is a navigational binding to a specific screen, construct in the TCA. I think I refactor it to recoveryPhraseDisplayViewBinding so it's easier for anyone to understand.